akka-sample-cluster-scala/README.md

```diff
@@ -1,179 +1,157 @@
 
 - Subscribe to cluster membership events
 - Sending messages to actors running on nodes in the cluster
 - Cluster aware routers
-- Cluster metrics
 
 ## A Simple Cluster Example
 
 Open [application.conf](src/main/resources/application.conf)
 
-To enable cluster capabilities in your Akka project you should, at a minimum, add the remote settings, and use `akka.cluster.ClusterActorRefProvider`. The `akka.cluster.seed-nodes` should normally also be added to your application.conf file.
+To enable cluster capabilities in your Akka project you should, at a minimum, add the remote settings, and use `cluster` as the `akka.actor.provider`. The `akka.cluster.seed-nodes` should normally also be added to your `application.conf` file.
 
 The seed nodes are configured contact points which newly started nodes will try to connect with in order to join the cluster.
 
 Note that if you are going to start the nodes on different machines you need to specify the ip-addresses or host names of the machines in `application.conf` instead of `127.0.0.1`.
 
-Open [SimpleClusterApp.scala](src/main/scala/sample/cluster/simple/SimpleClusterApp.scala).
+Open [SimpleClusterApp.scala](src/main/scala/sample/cluster/simple/App.scala).
 
-The small program together with its configuration starts an ActorSystem with the Cluster enabled. It joins the cluster and starts an actor that logs some membership events. Take a look at the [SimpleClusterListener.scala](src/main/scala/sample/cluster/simple/SimpleClusterListener.scala) actor.
+The small program together with its configuration starts an ActorSystem with the Cluster enabled. It joins the cluster and starts an actor that logs some membership events. Take a look at the [SimpleClusterListener.scala](src/main/scala/sample/cluster/simple/ClusterListener.scala) actor.
 
-You can read more about the cluster concepts in the [documentation](http://doc.akka.io/docs/akka/2.6/scala/cluster-usage.html).
+You can read more about the cluster concepts in the [documentation](https://doc.akka.io/docs/akka/2.6/typed/cluster.html).
 
-To run this sample, type `sbt "runMain sample.cluster.simple.SimpleClusterApp"` if it is not already started.
+To run this sample, type `sbt "runMain sample.cluster.simple.App"`.
 
-`SimpleClusterApp` starts three actor systems (cluster members) in the same JVM process. It can be more interesting to run them in separate processes. Stop the application and then open three terminal windows.
+`sample.cluster.simple.App` starts three actor systems (cluster members) in the same JVM process. It can be more interesting to run them in separate processes. Stop the application and then open three terminal windows.
 
 In the first terminal window, start the first seed node with the following command:
 
-    sbt "runMain sample.cluster.simple.SimpleClusterApp 2551"
+    sbt "runMain sample.cluster.simple.App 25251"
```

Comment:
triple quote around those?

Reply:
Why would that be needed?

Reply:
because I don't know markdown enough ;-)
strike that, the indentation seems to do the job